tests/unit/plugins/modules/test_snmp_manager.py

```diff
@@ -0,0 +1,285 @@
+# Copyright: (c) 2024, Dell Technologies
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""Unit Tests for storage container module for PowerStore"""
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+
+import pytest
+# pylint: disable=unused-import
+from ansible_collections.dellemc.powerstore.tests.unit.plugins.module_utils.libraries import initial_mock
+from mock.mock import MagicMock
+from ansible_collections.dellemc.powerstore.tests.unit.plugins.module_utils.mock_snmp_manager_api import MockSNMPManagerApi
+from ansible_collections.dellemc.powerstore.tests.unit.plugins.module_utils.mock_api_exception \
+    import MockApiException
+from ansible_collections.dellemc.powerstore.plugins.module_utils.storage.dell \
+    import utils
+from ansible_collections.dellemc.powerstore.tests.unit.plugins.module_utils.libraries.powerstore_unit_base \
+    import PowerStoreUnitBase
+from ansible_collections.dellemc.powerstore.plugins.modules.snmp_manager import PowerStoreSNMPManager, SNMPManagerHandler
+
+
+class TestPowerStoreSNMPManager(PowerStoreUnitBase):
+
+    get_module_args = MockSNMPManagerApi.SNMP_MANAGER_COMMON_ARGS
+
+    @pytest.fixture
+    def module_object(self):
+        return PowerStoreSNMPManager
+
+    def test_get_snmp_manager(self, powerstore_module_mock):
+        self.set_module_params(
+            powerstore_module_mock, self.get_module_args,
+            {
+                "ip_address": "127.0.0.1"
```

Comment:
Check for Sonar and checkmarx issues.

Reply:
this is dummy IP so sonar is not giving any issue with this